pandas/core/generic.py

```diff
@@ -6608,7 +6608,7 @@ def replace(
         inplace: bool_t = False,
         limit: int | None = None,
         regex=False,
-        method=lib.no_default,
+        method: str | lib.NoDefault = lib.no_default,
```

Comment:
```suggestion
        method: Literal["pad", "ffill", "bfill"] | lib.NoDefault = lib.no_default,
```

I think those are the possible methods

Reply:
Thank you! `None` seems also to be accepted based on the doc-string.

Reply:
I removed `None` from the documentation - it raises errors when using it.